_posts/2025-1-28-Arm-Cortex-FPU.md

```
@@ -51,24 +51,24 @@ This ensures smoother transitions around zero and allows representation of value
 - Bias for single precision = 127  
-- Exponent = 2 → `2 + 127 = 129`  
+- Exponent = 2 -> `2 + 127 = 129`  
 - `129` in binary = `10000001`
 
 #### Step 4: Mantissa (23 bits)  
 Keep the fraction part after the `1.` (since `1.` is implicit in normalized form)  
-`1.11` → take `.11` → `11000000000000000000000`
+`1.11` -> take `.11` -> `11000000000000000000000`
 
-#### ✅ Final IEEE 754 Format
+####  Final IEEE 754 Format
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)